toolchain/sem_ir/formatter.cpp

```diff
@@ -814,6 +825,11 @@ class Formatter {
     FormatLabel(inst.block_id);
   }
 
+  auto FormatInstructionRHS(LazyImportRef inst) -> void {
+    // Don't format the inst_id because it refers to a different IR.
```

Comment:
This is unfortunate. But I can't think of a better approach.

Worth a TODO to revisit this?

Also, maybe move this function next to the one for `CrossRef` and make them use the same formatting (I don't mind which one you choose).

Reply:
> This is unfortunate. But I can't think of a better approach.
> 
> Worth a TODO to revisit this?

Added a TODO.

> Also, maybe move this function next to the one for `CrossRef` and make them use the same formatting (I don't mind which one you choose).

Hah, I'd missed that. I found some overloads of `FormatInstructionRHS` and didn't keep looking for more. Maybe there's something to be done to improve the ordering of functions so that overloads of `FormatInstruction` variants are adjacent, or maybe moving the format to the typed instructions? (and also moving `Indent` out of the middle)